### PR TITLE
chore(package): update deps

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #3355 
Closes #3354 
Closes #3353 
Closes #3352 
Closes #3351 
Closes #3350

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,19 +26,19 @@
...
+    "eslint": "^7.21.0",
+    "eslint-config-prettier": "^8.1.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jasmine": "^4.1.2",
     "eslint-plugin-prettier": "^3.3.1",
     "hyperhtml": "^2.34.0",
     "idb": "^6.0.0",
     "jasmine": "^3.6.4",
-    "jquery": "^3.5.1",
+    "jquery": "^3.6.0",
```
Comment (sidvishnoi) [inline package.json:41]: why do we still have this here?

**Review by sidvishnoi — APPROVED** [review]

File: tests/spec/core/data-include-spec.js [hunk 2]
```diff
@@ -135,18 +135,19 @@ describe("Core — Data Include", () => {
...
       expect(li).toBeTruthy();
-      expect(li.textContent).toContain("Rose");
-      expect(li.textContent).toContain("Blue");
-      expect(li.textContent).toContain("Red");
-      expect(li.textContent).toContain("wasteland");
+      expect(li.textContent).toBe("Rose");
+      const [p1, p2, p3] = [...doc.querySelectorAll("#test p")];
+      expect(p1.textContent).toContain("Blue");
+      expect(p2.textContent).toContain("Red");
+      expect(p3.textContent).toContain("wasteland");
```
Comment (sidvishnoi) [inline tests/spec/core/data-include-spec.js:150]: See https://github.com/markedjs/marked/issues/1934 (and the fix: https://github.com/markedjs/marked/pull/1936)
cc: @saschanaz

Comment (saschanaz) [reply tests/spec/core/data-include-spec.js:150]: The intention here was to wrap them to `li`. It seems marked now requires at least 3-space indentation here.

Comment (sidvishnoi) [reply tests/spec/core/data-include-spec.js]: 3-space fixes it. Shouldn't be a breaking change for us as we've delegated markdown related changes on marked, which is commonmark compliant (almost).
Thanks!

**Pull request merged**